gcp/modules/gcp-stackdriver-alerting/main.tf

```diff
@@ -7,6 +7,10 @@ variable "domain_name" {}
 variable "project_id" {}
 variable "serviceaccount_key" {}
 
+variable "stackdriver_debug" {
```

Comment:
Hmm, this variable is never used. I guess we can delete it and rely only on `ENV["TF_VAR_stackdriver_debug"]`? (We need to keep the `TF_VAR_` prefix so that the variable is passed from the user's environment through to processes running in the exekube container.)

Reply:
Well, the same way we can get rid of `PROJECT_ID` and `GOOGLE_CLOUD_KEYFILE` vars and just use `TF_VAR_project_id`, `TF_VAR_serviceaccount_key` in the Ruby code. But I don't really like that – it may be hard to keep track of all ENV vars in use by client without them being declared in Terraform module. I'll keep that var and add `export STACKDRIVER_DEBUG=${var.stackdriver_debug}` before Ruby call instead.